pineappl/src/grid.rs

```diff
@@ -2858,24 +2874,58 @@ mod tests {
         assert_eq!(
             results,
             [
-                877.9498750860583,
-                823.5123400865052,
-                735.605093586326,
-                616.3465722662226,
-                478.63703336207277,
-                341.06729874517384,
-                174.3688634669724,
-                48.27440593682665
+                1469.0819829706113,
+                4364.509970805006,
+                2072.647269805865,
+                742.6160478667135,
+                312.195449854513,
+                145.782031788469,
+                72.64484245290579,
+                38.88392569051013,
+                17.093513927247354,
+                6.157887576517187,
+                2.4621639421455903,
+                1.0817834362838417,
+                0.5084171526510098,
+                0.2520459057372801,
+                0.10211930488819178,
+                0.021141855492915994
             ]
         );
 
+        //assert_eq!(
+        //    results,
+        //    [
+        //        134632.4480167966,
+        //        133472.53856965082,
+        //        130776.9197998554,
+        //        126991.17650299768,
+        //        121168.15135839234,
+        //        112761.75563082621,
+        //        98484.12455342771,
+        //        57843.82009866679
+        //    ]
+        //);
+
+        //assert_eq!(
+        //    results,
+        //    [
+        //        877.9498750860583,
+        //        823.5123400865052,
+        //        735.605093586326,
+        //        616.3465722662226,
+        //        478.63703336207277,
+        //        341.06729874517384,
+        //        174.3688634669724,
+        //        48.27440593682665
+        //    ]
+        //);
+
```

Comment:
The best option would be to store these values alongside the grid (such that we don't need to remember which correspond to which, and uncomment at the same time), most likely one folder for each grid, and upload them somewhere.

I'm not sure we want them in the repo, but maybe we can just upload them to the NNPDF data server (and access it with a secret, for running in the CI).

Reply:
Agreed, for the time being it's just there so I can quickly test.

Reply:
@AleCandido we definitely need this, also to close #122. Is there any server that we can use to quickly implement this? We could pair this with <https://github.com/actions/cache> to not always download all the stuff.

Reply:
This is (at least for the time being) fixed in commits f91e7cab61bd8327401ecdb9f8a516c87de79be8 and 5f485fff0a4ee71c7b17e5cbb1202ce13bcb1860, which download the data from our NNPDF data server. In the future we might want to change this, for the time being it's sufficient.